client/state/sites/plans/actions.js

```diff
@@ -127,7 +127,7 @@ export function fetchSitePlansCompleted( siteId, data ) {
 	let plansData = omit( data, '_headers' );
 	const { product_id } = personalPlan;
 
-	if ( ! has( plansData, product_id ) ) {
+	if ( ! isEmpty( plansData ) && ! plansData.hasOwnProperty( product_id ) ) {
```

Comment:
hmmm…I think I may have misread the logic. we want to run the conditional block _if and only if_  the data is both not-empty _and_ does not have the product in it. inversely (de Morgan's law), we want to _skip the block_ if the data is either empty _or_ if it has the plan in it. is that right?

```js
if ( ! ( isEmpty( plansData ) || has( plansData, product_id ) )  {
	…
}
```

I think I had trouble reading the triple-negative in the condition